torch/optim/nadam.py

```diff
@@ -0,0 +1,111 @@
+import torch
+from . import _functional as F
+from .optimizer import Optimizer
+
+
+class NAdam(Optimizer):
+    r"""Implements NAdam algorithm.
+
+    It has been proposed in `Incorporating Nesterov Momentum into Adam`_.
+
+    Args:
+        params (iterable): iterable of parameters to optimize or dicts defining
+            parameter groups
+        lr (float, optional): learning rate (default: 2e-3)
+        betas (Tuple[float, float], optional): coefficients used for computing
+            running averages of gradient and its square (default: (0.9, 0.999))
+        eps (float, optional): term added to the denominator to improve
+            numerical stability (default: 1e-8)
+        weight_decay (float, optional): weight decay (L2 penalty) (default: 0)
+        schedule_decay (float, optional): momentum schedule decay (default: 4e-3)
```

Comment:
would the name `momentum_decay` be more appropriate? `schedule` is overloaded.

Reply:
where was the default valued for this picked from?

Reply:
I again agree that there is no clear way for the default value in the research paper, however, i believe this is the common choice in the practice and again Keras using the same notation default value : 
https://github.com/tensorflow/tensorflow/blob/f9d386849581d15d72f6f1f96f12aac230a8edbe/tensorflow/python/keras/optimizer_v2/nadam.py#L72

but i am more than happy to adjust name/default if you have a suggestion :)

Reply:
please add this reference to the description of the pr

Reply:
its done ;)

Reply:
I recommend being very explicit with the name and use `momentum_schedule_decay` or `momentum_decay` instead. I prefer the later since it follows the convention we have here with `weight_decay`, even though the former is a more explicit reference to tensorflow.